DOKS-wordpress/README.md

```diff
@@ -390,13 +410,66 @@ wordpress.local-tls   True    wordpress.local-tls   24h
 
 Now, you can access the WordPress using the domain configured earlier.
 
-### Confiugre plugins
+### Enable WordPress metrics
+
+In this section, you will learn how to enable the WordPress metrics for monitoring your cluster.
+
+First, open the YAML file `(values.yml)` created earlier and enable the metrics, like in the below example:
+
+```yaml
+# Prometheus Exporter / Metrics configuration
+metrics:
+  enabled: true
+```
+
+Apply the change via `helm`:
+
+```console
+helm upgrade wordpress bitnami/wordpress \
+    --create-namespace \
+    --namespace wordpress \
+    --version 13.1.4 \
+    --timeout 10m0s \
+    --values values.yml
+```
+
+Next, please perform a `port-forward`, to inspect the metrics:
+
+```console
+kubectl port-forward --namespace wordpress svc/wordpress-metrics 9150:9150
+```
+
+The exposed `metrics` can be `visualized` using the web browser on localhost:
+
+```url
+http://127.0.0.1:9150/metrics
+```
+
+Finally, you need to configure on your cluster the Grafana and Prometheus.
```

Comment:
Maybe:

>Finally, you need to configure Grafana and Prometheus to visualise metrics exposed by your new WordPress instance.

Shouldn't we add a link here on how to accomplish the above  (maybe pointing to the Starter Kit, or any other DO documentation for that matter) ?